core/src/main/java/org/apache/cxf/helpers/DOMUtils.java

```diff
@@ -790,7 +790,10 @@ public static DocumentFragment getDomDocumentFragment(DocumentFragment fragment)
             //java9 plus hack
             Field f = GET_DOCUMENT_FRAGMENT_FIELDS.get(fragment.getClass());
             if (f != null) {
-                return ReflectionUtil.accessDeclaredField(f, fragment, DocumentFragment.class);
+                final DocumentFragment inner = ReflectionUtil.accessDeclaredField(f, fragment, DocumentFragment.class);
```

Comment:
I am not able to reproduce the issue though but it seems like to be the only place where `null` could escape